src/utils.jl

```diff
@@ -7,10 +7,10 @@ reduction, all elements are rounded to `digits` after the decimal. Rows of all z
 (absolute value of the each element in row ≤ `atol`) are removed. Beware, this
 operation is expensive for very large matrices.
 """
-function _remove_lin_dep_rows(A; ϵ = 1e-8, atol = 1e-8, digits = 16)
+function _remove_lin_dep_rows(A; ϵ = 1e-8, atol = 1e-8)
     #TODO this method is suboptimal and can be improved for numerical stability
     #TODO improve RowEchelon, SVD does not work due to column reordering
-    rA = round.(rref!(copy(Array(A)), ϵ); digits)
+    rA = rref!(copy(Array(A)), ϵ)
```

Comment:
For most types, `Array(x)` already creates a copy, if A is quite large it can be worth checking but the additional copy is not needed I believe